modules/proto/rollkitsdk/sequencer/v1/query.proto

```
@@ -0,0 +1,41 @@
+syntax = "proto3";
+package rollkitsdk.sequencer.v1;
+
+import "amino/amino.proto";
```

Buf (1.54.0)

import "amino/amino.proto": file does not exist

(COMPILE)